src/main.rs

```diff
@@ -51,12 +51,18 @@ pub fn proof_from_file<T: for<'a> Deserialize<'a>>(proof_path: &str) -> T {
     let proof: T = bincode::deserialize(buffer.as_slice()).unwrap();
     proof
 }
+fn get_scheduler_key_for_batch(batch_number: u64) -> &'static [u8] {
+    match batch_number {
+        1..=174710 => include_bytes!("keys/verification_scheduler_key.json"),
+        _ => include_bytes!("keys/verification_scheduler_key_174710.json"),
```

Comment:
could we name it 'v5'? (to be consistent with circuit naming?)

Reply:
good point updated.
FYI: `node + leaf commitment update` might not result in `scheulder vk` update, so in scheduler we might have `v5`, `v9` etc, i.e not sequential order.